hermes-console-vue/src/views/admin/readiness/ReadinessView.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+  import { useI18n } from 'vue-i18n';
+  import { useReadiness } from '@/composables/use-readiness/useReadiness';
+  import ConsoleAlert from '@/components/console-alert/ConsoleAlert.vue';
+  import LoadingSpinner from '@/components/loading-spinner/LoadingSpinner.vue';
+  import ReadinessBreadcrumbs from '@/views/admin/readiness/readiness-breadcrumbs/ReadinessBreadcrumbs.vue';
+
+  const { t } = useI18n();
+
+  const { datacentersReadiness, loading, error } = useReadiness();
+</script>
+
+<template>
+  <v-container fill-height fluid class="mx-auto">
+    <v-row dense>
+      <v-col md="12">
+        <readiness-breadcrumbs></readiness-breadcrumbs>
```

Comment:
```suggestion
        <readiness-breadcrumbs />
```
I guess we should prefer self-closing tags where possible

(btw just googled and found out they are now legal even in Angular, since version 15 - unbelievable😮)